src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  max-width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(/src/images/imac.jpeg);
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin-top: 33px;
+    margin-left: 20px;
+  }
+
+  &__name {
+    width: 166px;
+    height: 36px;
+    padding: 40px 16px 0;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    padding: 4px 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &:hover {
+    border: 1px solid blue;
+    border-radius: 5px;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 24px 16px 0;
+  }
+
+  &__price-key {
+    width: 31px;
+    height: 18px;
+
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__price-value {
+    width: 50px;
+    height: 18px;
+
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: #060b35;
+  }
```

Comment:
The `&__price-value` element should have its styles, such as width, height, font size, and color, checked against the design specifications and updated if there are discrepancies.